<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTextDialog</class>
 <widget class="QDialog" name="AddTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabLanguages">
      <attribute name="title">
       <string>Languages</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollLanguages">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollContentLanguages">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>571</width>
            <height>276</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelID">
             <property name="text">
              <string>Unique ID :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineID"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConnections">
      <attribute name="title">
       <string>Connections</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollConnection">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollContentConnection">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>638</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="next" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="labelNext">
                <property name="text">
                 <string>Next :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboNext">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="prev" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="labelPrev">
                <property name="text">
                 <string>Previous :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QListWidget" name="listPrev">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                   </property>
                   <property name="movement">
                    <enum>QListView::Free</enum>
                   </property>
                   <property name="flow">
                    <enum>QListView::TopToBottom</enum>
                   </property>
                   <property name="resizeMode">
                    <enum>QListView::Fixed</enum>
                   </property>
                   <property name="layoutMode">
                    <enum>QListView::SinglePass</enum>
                   </property>
                   <property name="viewMode">
                    <enum>QListView::IconMode</enum>
                   </property>
                   <property name="uniformItemSizes">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                   <property name="sortingEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_9" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QPushButton" name="addPrev">
                      <property name="text">
                       <string>Add</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="removePrev">
                      <property name="text">
                       <string>Remove</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="parent" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelParent">
                <property name="text">
                 <string>Parents :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QListWidget" name="listParent">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                   </property>
                   <property name="movement">
                    <enum>QListView::Free</enum>
                   </property>
                   <property name="flow">
                    <enum>QListView::TopToBottom</enum>
                   </property>
                   <property name="resizeMode">
                    <enum>QListView::Fixed</enum>
                   </property>
                   <property name="layoutMode">
                    <enum>QListView::SinglePass</enum>
                   </property>
                   <property name="viewMode">
                    <enum>QListView::IconMode</enum>
                   </property>
                   <property name="uniformItemSizes">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                   <property name="sortingEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="addParent">
                      <property name="text">
                       <string>Add</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="removeParent">
                      <property name="text">
                       <string>Remove</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="son" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelSon">
                <property name="text">
                 <string>Sons :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QListWidget" name="listSon">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                   </property>
                   <property name="movement">
                    <enum>QListView::Free</enum>
                   </property>
                   <property name="flow">
                    <enum>QListView::TopToBottom</enum>
                   </property>
                   <property name="resizeMode">
                    <enum>QListView::Fixed</enum>
                   </property>
                   <property name="layoutMode">
                    <enum>QListView::SinglePass</enum>
                   </property>
                   <property name="viewMode">
                    <enum>QListView::IconMode</enum>
                   </property>
                   <property name="uniformItemSizes">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                   <property name="sortingEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QPushButton" name="addSon">
                      <property name="text">
                       <string>Add</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="removeSon">
                      <property name="text">
                       <string>Remove</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="labelError">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
